pkg/ctl/delete/iamserviceaccount.go

```diff
@@ -0,0 +1,132 @@
+package delete
+
+import (
+	"fmt"
+
+	"github.com/kris-nova/logger"
+	"github.com/spf13/pflag"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/kubernetes"
+	"github.com/weaveworks/eksctl/pkg/printers"
+)
+
+func deleteIAMServiceAccountCmd(cmd *cmdutils.Cmd) {
+	cfg := api.NewClusterConfig()
+	cmd.ClusterConfig = cfg
+
+	serviceAccount := &api.ClusterIAMServiceAccount{}
+
+	cfg.IAM.WithOIDC = api.Enabled()
+	cfg.IAM.ServiceAccounts = append(cfg.IAM.ServiceAccounts, serviceAccount)
+
+	var onlyMissing bool
+
+	cmd.SetDescription("iamserviceaccount", "Create an iamserviceaccount - AWS IAM role bound to a Kubernetes service account", "")
```

Comment:
Shouldn't the description say "delete an iam service account" instead of create?